src/util/file_util.cpp

```diff
@@ -61,11 +61,12 @@ std::string get_current_working_directory()
   std::string working_directory=wd;
   free(wd);
   #else
-  char buffer[4096];
+  wchar_t buffer[4096];
   DWORD retval=GetCurrentDirectory(4096, buffer);
```

Comment:
Just checking: was Visual Studio now happy with `4096` or did it still complain about a buffer overrun?

Reply:
It is happy with 4096, I imagine it's number of character rather than size in bytes